### PR TITLE
release: 2.59.4

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

**Review by Meulengracht — APPROVED** [review]
LGTM!

File: packaging/debian-sid/changelog [hunk 1]
```diff
@@ -1,3 +1,11 @@
+snapd (2.59.4-1) unstable; urgency=medium
+
+  * New upstream release, LP: #2009946
+    - Retry when looking label on non-UEFI systems
```
Comment (alfonsosanchezbeato) [inline packaging/debian-sid/changelog:4]: I think this should mention https://bugs.launchpad.net/snapd/+bug/2018977

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# New in snapd 2.59.4:
+* Retry when looking label on non-UEFI systems
```
Comment (pedronis) [inline NEWS.md:2]: shouldn't this say disk label?

**Review by pedronis — APPROVED** [review]
one comment

File: packaging/debian-sid/changelog (hunk 1 above)
Comment (mvo5) [reply packaging/debian-sid/changelog:13]: I updated this to include the LP bug now

File: NEWS.md (hunk 2 above)
Comment (mvo5) [reply NEWS.md]: Good point, I updated this too

**Review by alfonsosanchezbeato — APPROVED** [review]
Thanks!

**Pull request merged**